src/main/java/com.gigssandbox/ConsoleHelper.java

```diff
@@ -1,18 +1,22 @@
 package com.gigssandbox;
 
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.util.Collection;
 import java.util.Properties;
 import java.util.Scanner;
 
 public class ConsoleHelper {
 
-
     private static final Scanner in = new Scanner(System.in);
-    private Properties properties;
-
+    private Properties properties = new Properties();
 
-    public ConsoleHelper(Properties properties) {
-        this.properties = properties;
+    public ConsoleHelper() {
+        try {
+            properties.load(new FileInputStream("src/main/resources/strings.properties"));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
What if you catch this exception? Not sure that further interaction with user is going to be successful. I think this is developer issue and here we can throw some kind of unchecked IllegalStateException